backend/src/Designer/Controllers/PreviewController.cs

```diff
@@ -304,6 +306,18 @@ public async Task<ActionResult> GetFormData(string org, string app, string curre
             return Ok(formData);
         }
 
+        /// <summary>
+        /// Action for updating the json schema for the datamodel for the default datatask test-datatask-id
+        /// </summary>
+        /// <remarks>Only for apps that does not use layoutsets. Must be adapted</remarks>
+        /// <returns>Json schema for datamodel for datatask test-datatask-id</returns>
+        [HttpPut]
+        [Route("instances/undefined/data/test-datatask-id")]
```

Comment:
I don't think this is the way we should do it long-term. We need to update the mock responses so that app-frontend actually has an instance-ID to use, instead of getting `undefined`. But as a quick-fix this does work for now.